Replace homepage screenshots (#208)
PrismLauncher/prismlauncher.org: src/index.md

      <div>
        <h1>Lightweight & Customizable</h1>
        <div class="subtitle">

>           You do not need an embedded web browser as a game launcher. Prism Launcher uses the Qt toolkit, which requires little system resources. Prism Launcher also let you customize the menu and supports [custom themes](https://prismlauncher.org/wiki/getting-started/change-themes/).

Reviewer: ```suggestion
          You don't need an embedded web browser as a game launcher. Prism Launcher uses the Qt toolkit, which requires little system resources. Prism Launcher also lets you customize the menu and supports [custom themes](https://prismlauncher.org/wiki/getting-started/change-themes/).
```
